src/main/java/xbot/common/controls/actuators/XCANMotorController.java

```diff
@@ -126,6 +130,8 @@ protected XCANMotorController(
         police.registerDevice(DevicePolice.DeviceType.CAN, busId, info.deviceId(), info.name());
         this.akitName = info.name()+"CANMotorController";
 
+        this.unhealthyAlert = new Alert("Motor Controller " + info.deviceId() + " on CAN bus " + busId.toString() +  " is unhealthy", Alert.AlertType.kError);
```

Comment:
⭐ ⭐  What do you think about including `owningSystemPrefix` and `pidPropertyPrefix` here so we don't have to go look these up later?